planty/users/models.py

```diff
@@ -1,3 +1,26 @@
 from django.db import models
+from django.contrib.auth.models import User
+from PIL import Image
+from planty.utils import crop_max_square
 
-# Create your models here.
+
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    image = models.ImageField(default='default.jpg',
+                              upload_to='profile_pictures')
```

Comment:
I see that `upload_to` is set to `profile_pictures` but your `default.jpg` image resides in `planty/media/default.jpg`. Does that work? Should it be in `planty/media/profile_pictures/default.jpg`

Reply:
Django looks for the default picture at:  media/default.jpg, so this works